### ISSUE TITLE
Use colons for error reporting for lines/char instead on brackets

### ISSUE THREAD

**Issue by orta** [palantir/tslint#3861]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.8.x
- __Running TSLint via__: CLI / VSCode

Links to the error can't be clicked in the terminal due to un-standard URLs

<img width="651" alt="screen shot 2018-04-28 at 5 13 19 pm" src="https://user-images.githubusercontent.com/49038/39400921-a3d76416-4b07-11e8-991d-f211d4c3ccb2.png">

I'd love to move them to use the format `file.ts:12:3` which matches how TypeScript shows these errors https://github.com/Microsoft/TypeScript/pull/20736 (and how test runners show it)

I'm happy to write this PR, but with 79 open PRs, I thought it'd be more prudent to do an issue before putting the time in 👯

#### TypeScript code being linted

n/a

#### Actual behavior

```sh
~/dev/projects/danger/peril plugins*
❯ yarn lint
yarn run v1.5.1
$  tslint source/**/**/**.ts

ERROR: source/api/_tests/fixtureAPI.ts[20, 1]: Use of default exports is forbidden
ERROR: source/api/pr/dsl.ts[62, 8]: Use of default exports is forbidden
ERROR: source/github/events/github_runner.ts[227, 8]: Use of default exports is forbidden

error An unexpected error occurred: "Command failed.
```

#### Expected behavior

```sh
~/dev/projects/danger/peril plugins*
❯ yarn lint
yarn run v1.5.1
$  tslint source/**/**/**.ts

ERROR: source/api/_tests/fixtureAPI.ts:20:1 - Use of default exports is forbidden
ERROR: source/api/pr/dsl.ts:62:8 - Use of default exports is forbidden
ERROR: source/github/events/github_runner.ts:227:8 - Use of default exports is forbidden

error An unexpected error occurred: "Command failed.
```

**Comment by giladgray** [comment]
@orta yes this is a great idea!
